addon/chrome/skin/app_version/78/linux/browser.css

```diff
@@ -7,12 +7,12 @@
 
 @import url("chrome://tabmixplus/skin/app_version/all/linux/browser.css");
 @import url("chrome://tabmixplus/skin/app_version/all/win/progress.css");
-@import url("chrome://tabmixplus/skin/app_version/39.0/tabStyles.css");
+@import url("chrome://tabmixplus/skin/app_version/86/tabStyles.css");
```

Comment:
why not 78 here ?

Reply:
I was trying use 86 first but as I state above it not yet come and with `appversion>=86` it seems asking for trouble.
This is a place I missing out will update this.